server/graphql/v1/TransactionInterface.js

```diff
@@ -236,85 +236,64 @@ const TransactionFields = () => {
     },
     host: {
       type: UserCollectiveType,
```

Comment:
Not from this PR, but taking note that this doesn't seem to be the best type. We probably won't fix it since it's GraphQL V1.